src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -168,7 +169,7 @@ private void ShowTermsOfUseForPublishing()
             if (resourceProvider != null)
                 additionalTerms = resourceProvider.AdditionalPackagePublisherTermsOfUse;
 
-            if (!ShowTermsOfUseDialog(true, additionalTerms))
+            if (!ShowTermsOfUseDialog(true, additionalTerms, WpfUtilities.FindUpVisualTree<DynamoView>(viewParent)))
```

Comment:
This also means that the check on Line 127 would fail to distinguish between the regular state and guided tour state, because the previous check was based on a wrong initial state. @RobertGlobant20 Is there a different check we can rely on to know if Guided tour is on-going?

Reply:
Not sure if this is useful but this flag isAnyGuideActive says if any guide is being executed.
https://github.com/DynamoDS/Dynamo/blob/644c0a0c8aa2ccc275185c9d2d4db584a5dfb887/src/DynamoCoreWpf/UI/GuidedTour/GuideFlowEvents.cs#L21